### ISSUE TITLE
Evaluation

### ISSUE THREAD

**Issue by OhataKenji** [OhataKenji/SketchRNN-Julia#5]
current evaluation on training data is to calculate the loss on random samples from training data

it should be the mean loss

**Comment by OhataKenji** [comment]
#12
